docker/generic/start_proxy.py

```diff
@@ -467,6 +471,47 @@ def make_argparser():
         - Debug HTTP response headers
         ''')
 
+    parser.add_argument(
+        '--transcoding_always_print_primitive_fields',
```

Comment:
Please fix the diff

Reply:
this is intentionally.  They were added in the deprecated flag section.